### PR TITLE
added capability to create index-pattern from configuration file

### PR DESCRIPTION
**Pull request by lzecca78**
Hi @everyone 👋 !
This PR wants to introduce the creation of kibana index-patterns directly from a configuration file. The reason of putting the HTTP post in the readinessProbe is because the service must be up and running and kibana takes minutes to do that, so there isn't other good alternative( probably one alternative could be a job, but the job should check the readiness of the kibana service, so could be an overengineering of the current implementation).
Well, if you have good alternative, please write down 👇  any further suggestions!

### CODE REVIEW

**Comment by nutellinoit** [comment]
Cool! What about injecting the index pattern in a `startupProbe` ?

**Comment by lzecca78** [comment]
@nutellinoit , yep probably is the best way to address it, but as you wrote is pretty new, for this reason, unless this new probe will be definitely stable and widely adopted I prefer to keep it in the readinessProbe. But for sure as soon as it will be introduced, is a super interesting refactoring!

**Review by ralgozino — APPROVED** [review]
I have mixed feelings about this approach, it's a clever hack but a hack nonetheless. But being that I can't think of a better way to do it, I'm approving.

**Review by lnovara — APPROVED** [review]
I'm in the same boat as @ralgozino, but since we lack a better approach you have my LGTM! 😄

**Comment by angelbarrera92** [comment]
LGTM! It would be nice if you add a simple release note into the unreleased file (if it does not exists, feel free to create it). Take a look to this PR as an example: https://github.com/sighupio/fury-kubernetes-opa/pull/25/files

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Pull request merged**
